buildroot/tests/BIGTREE_GTR_V1_0

```diff
@@ -10,7 +10,7 @@ restore_configs
 opt_set MOTHERBOARD BOARD_BTT_GTR_V1_0 SERIAL_PORT -1 \
         EXTRUDERS 8 TEMP_SENSOR_1 1 TEMP_SENSOR_2 1 TEMP_SENSOR_3 1 TEMP_SENSOR_4 1 TEMP_SENSOR_5 1 TEMP_SENSOR_6 1 TEMP_SENSOR_7 1
 # Not necessary to enable auto-fan for all extruders to hit problematic code paths
-opt_set E0_AUTO_FAN_PIN PC10 E1_AUTO_FAN_PIN PC11 E2_AUTO_FAN_PIN PC12 NEOPIXEL_PIN PF13 \
+opt_set E0_AUTO_FAN_PIN PC10 E1_AUTO_FAN_PIN PC11 E2_AUTO_FAN_PIN PC12 E3_AUTO_FAN_PIN -1 E4_AUTO_FAN_PIN -1 E5_AUTO_FAN_PIN -1 E6_AUTO_FAN_PIN -1 E7_AUTO_FAN_PIN -1 NEOPIXEL_PIN PF13 \
```

Comment:
I'm not sure how these `*_AUTO_FAN_PIN` numbers were chosen, but they are not real fan pins:
- `PC10` & `PC11` are in the EXP1 connector
- `PC12` is `E2_SERIAL_TX_PIN` / `E2_CS_PIN`